authentication-service/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+import com.saveourtool.save.buildutils.*
+
+plugins {
+    id("com.saveourtool.save.buildutils.kotlin-jvm-configuration")
+    id("com.saveourtool.save.buildutils.spring-boot-configuration")
+    id("com.saveourtool.save.buildutils.spring-data-configuration")
+    alias(libs.plugins.download)
```

Comment:
I'm not sure we need a download plugin here

Reply:
Removed